### PR TITLE
Fix linter errors

### PR DESCRIPTION
**Pull request by audrow**
This PR should fix linter errors raised from https://github.com/ament/ament_lint/pull/302.

### CODE REVIEW

**Comment by audrow** [comment]
Here is CI, which ran tests for packages above `ament_uncrustify`, including the failing packages in this repo (see [here](https://ci.ros2.org/job/ci_linux/13782/)). 
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/13789/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/8659/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/11499/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/13884/)

**Review by ahcorde — CHANGES_REQUESTED** [review]
the job is failing https://build.ros2.org/job/Rpr__geometry2__ubuntu_focal_amd64/202/

**Comment by audrow** [comment]
@ros-pull-request-builder retest please

**Comment by audrow** [comment]
@ros-pull-request-builder retest this please

File: tf2_ros/include/tf2_ros/message_filter.h [hunk 1]
```diff
@@ -686,7 +686,7 @@ class MessageFilter : public MessageFilterBase, public message_filters::SimpleFi
 
   static std::string stripSlash(const std::string & in)
   {
-    if (!in.empty() && (in [0] == '/')) {
+    if (!in.empty() && (in[0] == '/')) {
```
Comment (clalancette) [inline tf2_ros/include/tf2_ros/message_filter.h:689]: @audrow So the issue seems to be that the linter in the PR job wants this to be the old code, that is `in [0] == '/'`.  Does the new linter want it to be without the space?

**Comment by clalancette** [comment]
@ros-pull-request-builder retest this please

**Review by clalancette — APPROVED** [review]
This looks good to me.

Let's wait for the result of the Windows job (which I just kicked again) before merging this.

**Comment by clalancette** [comment]
The failures on Windows are well-known, so going ahead and merging this one.

**Pull request merged**